client/src/components/CommentCard.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import Card from 'react-bootstrap/Card';
+import {AiFillEdit } from 'react-icons/ai';
+import {MdDelete } from 'react-icons/md';
+import EditCommentPopup from './EditCommentPopUp';
+import StarDisplay from './StarDisplay';
+
+export default function CommentCard({ text, userName, date, rating, icon, commentId, commentUpdated }) {
+
+  const [show, setShow] = useState(false)
+
+  async function deleteComment(id) {
+
+    try {
+        const deletePost = await fetch(`/api/comment/${id}`, 
+        {method: "DELETE"})
+    } catch (error) {
+        console.error(error.message)
+    }
+
+    if(commentUpdated !== undefined) commentUpdated();
+  }
+
+
+  return (
+    <Card style={{ width: '48rem', display: 'flex', flexDirection: 'row', margin: '1rem' }}>
+      <div style={{ display: 'flex', alignItems: 'center', padding: '10px' }}>
+        <Card.Img 
+          style={{ width: '64px', height: '64px', borderRadius: '50%' }} 
+          src={"https://icon-library.com/images/no-user-image-icon/no-user-image-icon-3.jpg"} 
```

Comment:
Nothing to change. Just a factoid. If your image ends with `.jpg`, it won't have transparency so if you ever want transparency, look for an image that ends in `.png`

Reply:
Thank you! I didn't know this factoid! :)